packages/teleport/src/services/auth/auth.ts

```diff
@@ -31,36 +36,57 @@ const auth = {
     );
   },
 
-  login(userId, password, token) {
+  createRegisterChallengeWithToken(tokenId: string, deviceType: DeviceType) {
```

Comment:
Is there a method that creates a challenge without token? if not `createRegisterChallengeWithToken` -> `createRegisterChallenge`